music21/musicxml/test_xmlToM21.py

```diff
@@ -1328,12 +1328,13 @@ def testHiddenRests(self):
         # Voice 2: <forward> (half), quarter note, <forward> (quarter)
         s = converter.parse(testPrimitive.hiddenRests)
         v1, v2 = s.recurse().voices
-        self.assertEqual(v1.duration.quarterLength, v2.duration.quarterLength)
+        self.assertEqual(v1.duration.quarterLength, 4.0)
+        self.assertEqual(v2.duration.quarterLength, 3.0)
 
-        restV1 = v1.getElementsByClass(note.Rest)[0]
-        self.assertTrue(restV1.style.hideObjectOnPrint)
-        restsV2 = v2.getElementsByClass(note.Rest)
-        self.assertEqual([r.style.hideObjectOnPrint for r in restsV2], [True, True])
+        restsV1 = list(v1.getElementsByClass(note.Rest))
+        self.assertEqual(restsV1, [])
+        restsV2 = list(v2.getElementsByClass(note.Rest))
+        self.assertEqual(restsV2, [])
```

Comment:
note for @mscuthbert: this change implicitly tests that hidden rests are no longer created in documents with both Finale and something else (music21) in the software tags.

This is ~okay -- would be better to have a more explicit test for that.

Reply:
Done.